examples/playground/src/configuration.js

```diff
@@ -11,10 +11,10 @@ import GL from '@luma.gl/constants';
 
 import {registerLoaders} from '@loaders.gl/core';
 import {CSVLoader} from '@loaders.gl/csv';
-import {DracoLoader, DracoWorkerLoader} from '@loaders.gl/draco';
+import {DracoWorkerLoader} from '@loaders.gl/draco';
```

Comment:
@xintongxia An thought:

The loaders.gl 2.0 `DracoLoader` is now loading both worker and non-worker scripts dynamically, so there is no longer a massive bundle-size penalty to including it (the module size is down from ~3MB to ~11KB). 

Therefore, I think we could just make `@loaders.gl/3d-tiles` and `@loaders.gl/gltf` modules depend directly on `@loaders.gl/draco` instead of depending on a call to `registerLoaders` 

This would simplify for users, as applications would not longer need to keep track of when `DracoLoader` is needed and have to manually import and "inject" it
.

I think this would essentially be a non-breaking change (unless someone relies on Draco not being supported when not injected) so I think we could do it in a patch/minor update.




The point being that it might be more useful from